storm-core/src/jvm/org/apache/storm/utils/IFn.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.storm.utils;
+
+/**
+ * Created by rfarivar on 1/12/16.
+ */
+public interface IFn {
+    Object eval (Object obj);
+}
```

Comment:
Please delete this file and the mapVal function in Utils.  No IFn conflicts with clojure please it is too confusing and the mapVal function is not actually used so lets just remove it.